homeassistant/components/iqvia/__init__.py

```diff
@@ -58,17 +58,7 @@
 }
 
 CONFIG_SCHEMA = vol.Schema(
-    {
-        DOMAIN: vol.Schema(
-            {
-                vol.Required(CONF_ZIP_CODE): str,
-                vol.Required(CONF_MONITORED_CONDITIONS, default=list(SENSORS)): vol.All(
```

Comment:
I suggest we mark it as `cv.deprecated` so that the value is still accepted, but just isn't used anymore. That gives people time to remove it from their config.

Reply:
or just allow vol to remove that like the supervisor does?

Reply:
We shouldn't silently fix things. If the user sets a config option, we need to tell them if we are no longer honoring it.